targets/arduino_101/src/jerry-entry.c

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2016 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <stdarg.h>
+
+#include "jrt/jrt.h"
```

Comment:
IMHO, targets shouldn't include `jrt.h`. Please check `targets/default/jerry-port-default-io.c`. This file only needs two include files: `jerry-port.h` and `stdarg.h`